packages/frontend/src/components/Layout/NavigationSidebar.tsx

```diff
@@ -114,7 +111,13 @@ export default function NavigationSidebar() {
           closeDrawer={closeDrawer}
         />
       ))}
-      <DemoSidebarItem />
+      <Box position="fixed" bottom={2}>
```

Comment:
check the width of the nav buttons for all screen sizes pls

Reply:
for this do u mean like mobile view the hamburger menu disappears for `templates`?